orion-server/src/main/java/com/pinterest/orion/core/actions/aws/ReplaceEC2InstanceAction.java

```diff
@@ -262,6 +270,30 @@ protected RunInstancesResponse runInstance(Ec2Client ec2Client,
         logger().warning("IP conflict: " + ec2e + " , retry without using same address");
         runInstancesRequest = runInstancesRequest.toBuilder().privateIpAddress(null).build();
         runInstancesResponse = ec2Client.runInstances(runInstancesRequest);
+      } else if (ec2e.awsErrorDetails().errorCode().equals("InsufficientInstanceCapacity")
+          && containsAttribute(ATTR_FALLBACK_MAPPING)) {
+        String currentInstanceType = runInstancesRequest.instanceType().toString();
+        getResult().appendOut(
+            "Instance type " + currentInstanceType
+                + " not available, attempting to fallback to other instance types");
```

Comment:
Can this be moved down after the replacement type is figured out so that info can be logged too?
Can we also send a metric?

Reply:
Log has been moved down to include original + replacement instance type